app/models/ai_tool.rb

```diff
@@ -2,6 +2,7 @@
 
 class AiTool < ActiveRecord::Base
   validates :name, presence: true, length: { maximum: 100 }
+  validates :tool_name, presence: true, length: { maximum: 100 }, uniqueness: true
```

Comment:
tool_name should not be unique... it is a constraint on "persona" not on tool, the same tool name can exist multiple times with no problem